src/app.js

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios';
+
+async function getLand(land) {
```

Comment:
Probeer functienamen, variabelen en namen waar mogelijk in het Engels te formuleren

Reply:
is er niet ook het woord land in het Engels?...